src/App.jsx

```diff
@@ -2,30 +2,45 @@ import React, { Component } from 'react'
 import ListRecipes from './components/ListRecipes'
 import SingleRecipe from './components/SingleRecipe'
 import Login from './components/Login'
-import { Route, BrowserRouter, Switch } from 'react-router-dom'
+import { Route, Redirect } from 'react-router-dom'
```

Comment:
Can you explain this change?

Reply:
Yes, I was unable to make the props.history.push('/') work for re-direct and instead of spending more time solving that I just used Redirect instead. The BrowserRouter was a duplicate as it already existed in the index.js file. It accidentally got duplicated after pull from upstream. Switch caused me an issue with Redirect, had to remove it.